test/config_test.js

```diff
@@ -76,7 +76,7 @@ suite("config", function() {
       ]
     });
 
-    assume(cfg).deep.equals(undefined);
+    assume(cfg).deep.equals({});
```

Comment:
We could easily hack it to do the old thing... But I actually like this better.. And I doubt a major version upgrade is required to justify this...